src/helpers/Flickr/Flickr.js

```diff
@@ -0,0 +1,11 @@
+export const isValidFlickrId = (value) => {
```

Comment:
@danalvrz I would add a check here so that we throw a clear error if someone accidentally tries to use this on the server:
```
if (__SERVER__) {
    throw new Error('isValidFlickrId is only implemented on the client');
}
```